### PR TITLE
Bisq import fix

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: rotkehlchen/tests/utils/dataimport.py [hunk 1]
```diff
@@ -1181,13 +1181,15 @@ def assert_bisq_trades_import_results(rotki: Rotkehlchen):
...
         trade_type=TradeType.SELL,
-        amount=AssetAmount(FVal(2450)),
+        amount=AssetAmount(FVal(0.01)),
         rate=Price(FVal(49000.0000)),
         fee=Fee(FVal(0.29)),
         fee_currency=symbol_to_asset_or_token('BSQ'),
         link='',
         notes='ID: xxhee',
     )]
+    print(trades)
```
Comment (LefterisJP) [inline rotkehlchen/tests/utils/dataimport.py:1191]: Is this PR supposed to be ready? If yes wtf are 2 print statements doing here?

Comment (yabirgb) [reply rotkehlchen/tests/utils/dataimport.py]: My fault. Yes ready for review. Removed them

**Review by LefterisJP — APPROVED** [review]
LGTM but why did the e2e tests fail?

**Comment by yabirgb** [comment]
@LefterisJP Yes, as for what kelsos mentioned bugfixes didn't have the improvement in develop and querying balances was slow

**Review by LefterisJP — APPROVED** [review]
lgtm

**Pull request merged**